src/puppet.ts

```diff
@@ -1005,15 +1005,15 @@ export abstract class Puppet extends PuppetEventEmitter {
    * Room
    *
    */
-  public abstract roomAdd (roomId: string, contactId: string)          : Promise<void>
-  public abstract roomAvatar (roomId: string)                          : Promise<FileBox>
-  public abstract roomCreate (contactIdList: string[], topic?: string) : Promise<string>
-  public abstract roomDel (roomId: string, contactId: string)          : Promise<void>
-  public abstract roomList ()                                          : Promise<string[]>
-  public abstract roomQRCode (roomId: string)                          : Promise<string>
-  public abstract roomQuit (roomId: string)                            : Promise<void>
-  public abstract roomTopic (roomId: string)                           : Promise<string>
-  public abstract roomTopic (roomId: string, topic: string)            : Promise<void>
+  public abstract roomAdd (roomId: string, contactId: string, invite?: boolean) : Promise<void>
```

Comment:
Maybe `forceInvite` is better? What do you think @huan

Reply:
This is a super hard question!

Reply:
Any idea of this question? @huan